subsys/bootloader/bl_boot/bl_boot.c

```diff
@@ -105,6 +104,8 @@ void bl_boot(const struct fw_info *fw_info)
 	__set_MSP(vector_table[0]);
 	__set_PSP(0);
 
+	__ISB();
```

Comment:
There is one ISB above, after setting the CONTROL register.
Can you re-shuffle the code so you do not need to add two ISBs in one line?